interpreter/src/main/java/org/qbicc/interpreter/impl/VmInvokableImpl.java

```diff
@@ -132,7 +132,7 @@ public Object invokeAny(VmThread thread, VmObject target, List<Object> args) {
 
     Object run(VmThreadImpl thread, VmObject target, List<Object> args) {
         long invCnt = ((long) countHandle.getAndAdd(this, 1)) + 1;
-        if (invCnt == 100) {
+        if (invCnt == 1000) {
```

Comment:
Honestly you could convert this to an `if (false)`. It has taught us what we needed to know: we need a JIT. :-)

Reply:
Done...disabled it.